### PR TITLE
Add default values to SetKernelStatus parameters

### LINKED ISSUE #2019: Give default values for parameters, especially for SetKernelStatus
**Describe the flaw**
Documentation today often does not show default parameter values.

**Suggest an improvement**
Show default parameter values, especially in the docstring for `SetKernelStatus`.

### PR DESCRIPTION
**Pull request by jessica-mitchell**
This PR adds default values to the SetKernelStatus parameters
Note that  `min_update_time,  max_update_time, update_time_limit`  did not exist in the `GetKernelStatus` output.

Also added default value to `set_verbosity`

Fixes #2019

### CODE REVIEW

**Comment by morales-gregorio** [comment]
Hi @jessica-mitchell I would suggest not adding a default to the parameters marked as `read only`, since they cannot be set by the user directly

**Comment by jessica-mitchell** [comment]
@morales-gregorio Yah ok, thanks for pointing that out, I had my doubts about putting those values in. I'll remove them.

File: pynest/nest/lib/hl_api_info.py [hunk 1]
```diff
@@ -202,7 +202,7 @@ def set_verbosity(level):
 
     Parameters
     ----------
-    level : str
+    level : str, default: M_INFO
```
Comment (morales-gregorio) [inline pynest/nest/lib/hl_api_info.py:205]: I would writhe `default: 'M_INFO'`, with quotes

File: pynest/nest/lib/hl_api_simulation.py [hunk 2]
```diff
@@ -332,11 +333,11 @@ def SetKernelStatus(params):
         Defines all synapses which are plastic for the structural plasticity
         algorithm. Each entry in the dictionary is composed of a synapse model,
         the pre synaptic element and the postsynaptic element
-    structural_plasticity_update_interval : int
+    structural_plasticity_update_interval : int, default: 10000.0
```
Comment (morales-gregorio) [inline pynest/nest/lib/hl_api_simulation.py:336]: In my NEST version (2.20.0) this value defaults to 1000 not 10000, can you double check this value?

**Review by morales-gregorio — CHANGES_REQUESTED** [review]
I had a look at the values and besides two minor things they all look good! Also thanks for extending the descriptions with valid types and other extra information :)

File: pynest/nest/lib/hl_api_simulation.py (hunk 2 above)
Comment (jessica-mitchell) [reply pynest/nest/lib/hl_api_simulation.py:336]: This appears to be correct,  I get `'structural_plasticity_update_interval': 10000.0,`

File: pynest/nest/lib/hl_api_info.py (hunk 1 above)
Comment (jessica-mitchell) [inline pynest/nest/lib/hl_api_info.py:205]:
```suggestion
    level : str, default: 'M_INFO'
```

Comment (jessica-mitchell) [reply pynest/nest/lib/hl_api_info.py]: Done

File: pynest/nest/lib/hl_api_simulation.py (hunk 2 above)
Comment (morales-gregorio) [reply pynest/nest/lib/hl_api_simulation.py:336]: Alright! Then I guess it was changed in NEST 3

**Review by morales-gregorio — APPROVED** [review]

**Comment by morales-gregorio** [comment]
Looks ready to merge to me!

**Comment by jougs** [comment]
Many thanks for the contribution and the review.

**Pull request merged**